packages/create-amplify/src/tsconfig_initializer.ts

```diff
@@ -52,12 +53,26 @@ export class TsConfigInitializer {
       );
     }
 
+    let aggregatedStdout = '';
+    const aggregatorStream = new stream.Writable();
+    aggregatorStream._write = function (chunk, encoding, done) {
+      aggregatedStdout += chunk;
+      done();
+    };
+
     try {
-      await this.execa('npx', tscArgs, {
-        stdio: 'inherit',
+      const childProcess = this.execa('npx', tscArgs, {
+        stdin: 'inherit',
         cwd: this.projectRoot,
       });
+
+      childProcess?.stdout?.pipe(aggregatorStream);
+      childProcess?.stderr?.pipe(aggregatorStream);
+
+      await childProcess;
+      logger.debug(aggregatedStdout);
     } catch {
+      logger.debug(aggregatedStdout);
```

Comment:
This seems like a repeated pattern. We may want a function like `executeWithLogger` or something to abstract away this piping

Reply:
Done in [0735be9](https://github.com/aws-amplify/amplify-backend/pull/595/commits/0735be9988ffbeba2b82c2c3be25e57a1adae09e)